src/io.c

```diff
@@ -59,6 +59,9 @@
 **
 **  'number' is the number of the current line, is used in error messages.
 **
+**  'numberInterpreter' is the number of the line where the fragment of code
+**  currently being interpreted started, is used for profiling
```

Comment:
change the comma to a semicolon or period? I was at first slightly confused by reading it like that.

Reply:
Also, the corresponding GAPState member is named `InterpreterStartLine` -- why not name the two variables similarly? I find `InterpreterStartLine` much clearer than `numberInterpreter`.

Reply:
I named it numberInterpreter to line up with number (the previous variable), but I agree that's a silly name for a variable, and I shouldn't be carrying on it's usage :)